cli/pollapo/pollapoYml.ts

```diff
@@ -192,7 +192,18 @@ export async function* cacheDeps(
       const commitHash = await fetchingCommitHash;
       queue.push({ ...dep, rev: commitHash });
       await unlink(dep);
-      await link(dep, commitHash);
+      try {
+        await link(dep, commitHash);
+      } catch (err) {
+        if (Deno.build.os === "windows") {
+          console.log(
+            "Failed to symlink. Please run the command as administrator.",
+          );
+        } else {
+          console.log("Failed to symlink.");
```

Comment:
how about using console.error with color?
```suggestion
          console.error(red("Failed to symlink"))
```
you can import fmt/colors with 
```
import { red } from "https://deno.land/std@0.122.0/fmt/colors.ts";
```
we already use it in https://github.com/pbkit/pbkit/blob/3b3a42a7d5b535defbdf176e68e47cfe8b4a8a7e/cli/pollapo/cmds/why.ts#L91